Week-1/HTML Assignment1/index.html

```diff
@@ -0,0 +1,153 @@
+<html>
+  <head>
+	<link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <style>	
+		body {
+			font-family: 'Lato';
+		}
+      .bottomLabel {
+        color: white;
+      }
+      .bottomIcon {
+        padding-right: 100px;
+        height: 12px;
+        width: 12px;
+      }
+      .bottomIcon1 {
+        padding-left: 1000px;
```

Comment:
1000px? what is this for seems to be huge value

Reply:
updated with new units
 padding-left: 65em;